### DISCUSSION TITLE
Import Data Source for District

### DISCUSSION THREAD

**Discussion by sblack4** [code4recovery/12-step-meeting-list#1139] [Q&A] [answered]
Hi! District 28 recently set up this plugin (see [here](https://woodbridgeaa.org/meetings/)) and it's awesome. We are using the "Import Data Sources" to pull the meeting list from our parent region (https://aavirginia.org/meetings/). 

The issue we're having is that not all the meetings in our district are being imported. There doesn't seem to be a reliable `region` or `group_id` we can query on, and `region_id` returns the full set of meetings. 

The two data source feeds we've tried are 
- https://aavirginia.org/wp-admin/admin-ajax.php?action=meetings&group_id=161674 returns 52 meetings
- https://aavirginia.org/wp-admin/admin-ajax.php?action=meetings&region=woodbridge returns 51 meetings
- https://aavirginia.org/wp-admin/admin-ajax.php?action=meetings&region=woodbridge&group_id=161674 returns 49 meetings

The last one seems to do an AND between the two query params. Is there a way to do an OR? That would solve our problem. Also I'm not sure which of the fields should be for the district. I checked the [JSON spec](https://github.com/code4recovery/spec) and didn't see anything about districts. 

Thank you!

**Comment by joshreisner** [comment, marked as answer]
it's sort of undocumented but you can actually use `district` for this eg https://aavirginia.org/wp-admin/admin-ajax.php?action=meetings&district=district-28-woodbridge

**Reply by joshreisner** [reply]
it's awesome you checked the JSON spec - that describes the output you should be getting. there isn't really a place to document the inputs that the wp plugin listens for, but `district` is in there also `query` is for a search string, `day=0` (0 = sunday, 6 = saturday), basically whatever you see in the query string minus the `tsml-` part

**Reply by sblack4** [reply]
Thank you! I might be able to use the `query` for now but I think the right answer is to get the individual meetings updated so they have the right `district`
